### PR TITLE
[fix] send search request without sign.

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- send search requests with sign: `searchAinftObjects()`, `searchNfts()`

### CODE REVIEW

**Review by minjaeli — APPROVED** [review]
LGTM! thanks!

**Comment by jiyoung-an** [comment]
Thank you for quick review @minjaeli! now merging..

File: src/nft.ts [hunk 1]
```diff
@@ -201,8 +201,8 @@ export default class Nft extends FactoryBase {
       const { userAddress, ainftObjectId, name, symbol, slug, limit, cursor, order } = searchParams;
       query = { userAddress, ainftObjectId, name, symbol, slug, cursor, limit, order };
     }
-    const trailingUrl = `native/search/ainftObjects`;
-    return this.sendRequest(HttpMethod.GET, trailingUrl, query);
+    const trailingUrl = 'native/search/ainftObjects';
+    return this.sendRequestWithoutSign(HttpMethod.GET, trailingUrl, query);
```
Comment (platfowner) [inline src/nft.ts:205]: Could you leave some tsdoc comments about this?

File: src/nft.ts [hunk 2]
```diff
@@ -237,8 +237,8 @@ export default class Nft extends FactoryBase {
       const { userAddress, ainftObjectId, name, symbol, slug, tokenId, limit, cursor, order } = searchParams;
       query = { userAddress, ainftObjectId, name, symbol, slug, tokenId, limit, cursor, order };
     }
-    const trailingUrl = `native/search/nfts`;
-    return this.sendRequest(HttpMethod.GET, trailingUrl, query);
+    const trailingUrl = 'native/search/nfts';
+    return this.sendRequestWithoutSign(HttpMethod.GET, trailingUrl, query);
```
Comment (platfowner) [inline src/nft.ts:241]: ditto

File: src/nft.ts (hunk 1 above)
Comment (jiyoung-an) [reply src/nft.ts:205]: It's good idea! I'll make next pull request reflected this comment!

Comment (jiyoung-an) [reply src/nft.ts:205]: please check https://github.com/ainft-team/ainft-js/pull/311 PR.

**Pull request merged**